addons/pos_loyalty/static/src/js/Loyalty.js

```diff
@@ -185,18 +190,62 @@ const PosLoyaltyOrderline = (Orderline) => class PosLoyaltyOrderline extends Ord
                 }
             });
             for (const line of linesToRemove) {
-                this.order.orderlines.remove(line);
+                this.order.remove_orderline(line);
             }
         }
         return super.set_quantity(...arguments);
     }
+    /**
+     * @override
+     */
+    can_be_merged_with(otherLine) {
+        if (this.is_reward_line && otherLine.is_reward_line) {
+            if (otherLine.reward_id == this.reward_id) {
+                const reward = this.order.pos.reward_by_id[this.reward_id];
```

Comment:
```suggestion
                const reward = this.pos.reward_by_id[this.reward_id];
```

Reply:
I can't remember that Orderline's have reference to `pos`.

Reply:
yeah it does
```js
class Orderline extends PosModel {
    constructor(obj, options) {
        super(obj);
        this.pos   = options.pos;
```

Reply:
I see. Will do then.